### PR TITLE
gtk3-nocsd: provide a way for per-application integration

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

File: components/library/gtk3-nocsd/gtk3-nocsd.p5m [hunk 1]
```diff
@@ -22,9 +22,6 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=etc/X11/xinit/xinitrc.d/0310.gtk3-nocsd mode=0555 preserve=true
-
-file path=usr/bin/$(MACH64)/gtk3-nocsd
```
Comment (alarcher) [inline components/library/gtk3-nocsd/gtk3-nocsd.p5m:27]: Why no 64bit binary?

Comment (pyhalov) [reply components/library/gtk3-nocsd/gtk3-nocsd.p5m:27]: Because it's a script

Comment (alarcher) [reply components/library/gtk3-nocsd/gtk3-nocsd.p5m:27]: Will test today on my desktop, sorry for the lag.

Comment (pyhalov) [reply components/library/gtk3-nocsd/gtk3-nocsd.p5m:27]: Did you have chance to test it?

Comment (alarcher) [reply components/library/gtk3-nocsd/gtk3-nocsd.p5m:27]: I tested it and it seems to work fine. Opened devhelp, yelp, gucharmap: all looking good.

**Pull request merged**